### ISSUE TITLE
Add support for extensions

### ISSUE THREAD

**Issue by KrzysztofZawisla** [ridafkih/schemix#52]
https://www.prisma.io/docs/concepts/components/prisma-schema/postgresql-extensions

With the current implementation, this code:

```ts
datasource: {
  // rest
  extensions: `[pg_tgrm, zombodb]`,
  // rest
}
```

Evaluates to: 
 
 ```prisma
extensions = "[pg_tgrm, zombodb]"
```

**Comment by ridafkih** [comment]
@KrzysztofZawisla What if you provide an array of strings? Can add type support as well, but that *should* work unless I'm horribly wrong.

**Comment by ridafkih** [comment]
@KrzysztofZawisla Added to https://github.com/ridafkih/schemix/pull/53, if you'd like to test, otherwise I'll have this merged in tomorrow! :)

**Comment by ridafkih** [comment]
Has been merged and released in v1.8.4

**Comment by KrzysztofZawisla** [comment]
> @KrzysztofZawisla What if you provide an array of strings? Can add type support as well, but that _should_ work in the meantime unless I'm horribly wrong.

Sorry, I was out of programming vibe for these 4 days, so I didn't even check replies on this issue. It's important to not have items in array as string, because they will also be evaluated to string in prisma schema. Prisma requires passing extensions not as string to properly highlight syntax. In free time I will check the new release if it solves that problem.

**Comment by ridafkih** [comment]
@KrzysztofZawisla Thanks for letting me know! I'll make this adjustment tonight.

**Comment by ridafkih** [comment]
@KrzysztofZawisla Alright, #54 should address this fair and square. I was actually looking at the project you're contributing to the other day that implemented Schemix, and was impressed by the scale at which you guys took the library. Was very happy to see that. 

Since you guys are really using the library well, if you have any other suggestions, feature requests, or bug reports, let me know and I'll be sure to address them as soon as I can.

**Comment by ridafkih** [comment]
Released in [v1.8.5](https://github.com/ridafkih/schemix/releases/tag/v1.8.5) since the change does not touch any other components, let me know if that addresses that and I will close this issue.

**Comment by KrzysztofZawisla** [comment]
@ridafkih I have checked the new release (1.8.5) and everything works perfectly good.

I'm also happy about the fact that you have created that library, and we found it. Unique and well-done documented library in this field. Our Prisma schema expanded dramatically, and we thought that have to be separated + we had issues with stuff that must have done programmatically like country codes or enums generated based on typescript enums. We will make a PR or report issue in case when we will found a way to improvement something, but firstly we need to make our PostgreSQL, MongoDB, and FaunaDB schemas/structure and migrations 😅. 

Best Regards,
Krzysztof Zawisła and @borys-malinowski
